roles/dosomething.haproxy/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: Add HA Proxy PPA
+  become: yes
+  apt_repository: repo='ppa:vbernat/haproxy-1.5' update_cache=yes
+
+- name: Install HA Proxy
+  become: yes
+  apt: name=haproxy state=latest
+
+- name: Put Sysctl.conf Settings In Place
+  become: yes
+  sysctl: name={{ item.setting }} value={{ item.value }} sysctl_set=yes reload=yes
+  with_items:
```

Comment:
If you have just one variable to provide to `with_items` statement, you can omit providing it as a member of new list.
Just `with_items: sysctl_tweaks`